ethereum/gateway-eth-ts/.eslintrc

```diff
@@ -1,6 +1,7 @@
 {
   "root": true,
   "parser": "@typescript-eslint/parser",
+  "parserOptions": {"project": ["./tsconfig.json"]},
```

Comment:
This is needed? Seems like picking up the default tsconfig.json should not be required explicitly. But did not look up doc

Reply:
Yeah, "yarn lint" fails if you don't have the "parserOptions" option in, setting where the tsconfig is. It was a fix for it that I found on stack overflow.

Reply:
can you point me to that article?